bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -276,11 +282,13 @@ const BaseContainer = withTracker(() => {
     annotationsHandler,
     groupChatMessageHandler,
     userSubscriptionHandler,
+    userInfoSubscriptionHandler,
     breakoutRoomSubscriptionHandler,
     meetingModeratorSubscriptionHandler,
     animations,
     meetingExist: !!Meetings.find({ meetingId }).count(),
     User,
+    UserInfo,
```

Comment:
Why do we need to pass this here and not just obtain it in  bigbluebutton-html5/imports/ui/components/app/component.jsx from a service/container